### PR TITLE
NXBT-3289: Enrich Kubernetes Docker Secret with all authentications

### PR DESCRIPTION
**Pull request by doulba**

### CODE REVIEW

File: docs/platform-setup.md [hunk 1]
```diff
@@ -130,26 +130,28 @@ cluster:
 
 #### Create a Secret Based on Existing Docker Credentials
 
-We suppose you have a json file containing credentials for `docker.platform.dev.nuxeo.com`
+We suppose the Kubernetes secret `jenkins-docker-cfg` already exist in the `platform` namespace:
```
Comment (ataillefer) [inline docs/platform-setup.md:133]:
```suggestion
We suppose the Kubernetes secret `jenkins-docker-cfg` already exists in the `platform` namespace:
```

**Comment by ataillefer** [comment]
@kevinleturc we could, but we think that sticking with manual steps for all the secrets is OK for now, until we have something fully automated, hopefully with a vault...

And, even if we handle it in the Jenkinsfile, if we upgrade the Jenkins secret, the other one won't be updated until an env upgrade...

**Comment by doulba** [comment]
> Is it possible/wanted to create this secret during the JX installation/upgrade (in Jenkinsfile)?
> As it, we would have only one secret to manually add/edit. Wdyt?

In my opinion, this secret should be created at the beginning  (cluster and Jenkins first configuration).
in case of migration this question can be addressed again I think

**Comment by kevinleturc** [comment]
> And, even if we handle it in the Jenkinsfile, if we upgrade the Jenkins secret, the other one won't be updated until an env upgrade...
Agree with that, I was thinking about the cluster creation and introduction of the new nexus secret which could have had the benefit of this behavior as we also have changed the `jenkins-docker-cfg`.

Indeed now that everything is configured, we don't need such a thing and we can look forward 👍.

**Pull request merged**